src/services/Content/ContentApi.ts

```diff
@@ -330,12 +330,34 @@ export const introspectRepository: (
   return data;
 };
 
+export const triggerSnapshot: (
+  repositoryUUID: string,
+  resetCount: number,
+) => Promise<void> = async (repositoryUUID, resetCount) => {
+  try {
+    const { data } = await axios.post(
+      "'POST /api/content_sources/v1.0/repositories/$UUID/snapshot/'",
```

Comment:
this won't work as is because you've got single quotes in the path, as well as POST.  POST is an http method that is covered by the axios.post() function you are calling:  https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods    Take a look at the introspectRepository method: https://github.com/content-services/content-sources-frontend/blob/main/src/services/Content/ContentApi.ts#L323-L331  you basically need to do exactly what its doing but with '/api/content_sources/v1.0/repositories/$UUID/snapshot/'  

Also you don't need resetCount, as its not a valid parameter for this api